test/test_torch.py

```diff
@@ -12001,6 +12001,33 @@ def logcumsumexp(a, axis):
                 'expected scalar_type Double but found Float'):
             torch.logcumsumexp(b, axis, out=inplace_out)
 
+    def _test_large_cum_fn_helper(self, x, fn):
+        print(x.min(), x.max())
+        x_cpu = x.cpu().float()
+        expected = fn(x_cpu)
+        actual = fn(x).cpu().float()
+        self.assertEqual(expected, actual.cpu().float())
```

Comment:
```suggestion
        self.assertEqual(expected, actual)
```

Reply:
I always forget whether assertEqual can handle different devices and dtypes

Reply:
You are already doing `.cpu().float()` in the line above, so no need to do it here again.